src/docs/santiment-queries/metric-tables/index.md

```diff
@@ -0,0 +1,374 @@
+---
+title: Metrics Datasets
+author: Santiment Team
+date: 2024-04-01
+description: Learn about Santiment metrics datasets.
+---
+
+## Overview
+
+This document introduces Santiment's precomputed metrics datasets.
+
+You can filter all metric tables as:
+
+```sql
+SHOW TABLES LIKE '%metric%'
+```
+
+Test in [Queries](https://queries.santiment.net/query/list-of-tables-390)
+
+Usually, tables storing pre-computed data have a common set of columns:
+
+- `dt` - A `DateTime` field storing the corresponding date and time.
+- `asset_id` - An `UInt64` unique identifier for an asset. The data for that ID is stored in the `asset_metadata` table.
+- `metric_id` - An `UInt64` unique identifier for a metric. The data for that ID is stored in the `metric_metadata` table.
+- `value` - A `Float` column holding the metric's value for the given asset/metric pair.
+- `computed_at` - A `DateTime` column storing the date and time when the given row was computed.
+
+Metrics are located in a different tables based on different parameters. All tables are similar to each other and described below.
```

Comment:
```suggestion
Metrics are located in different tables, depending on some different parameters. All tables are similar to each other and described below.
```
We should give examples. I was about to say that `intraday` contains intraday metrics and `daily` contains daily metrics, but there are a lot of daily metrics in intraday.